content/docs/command-reference/exp/run.md

```diff
@@ -195,29 +207,85 @@ experiment we just ran (`exp-44136`).
 ## Example: Modify parameters on-the-fly
 
 `dvc exp run --set-param` (`-S`) saves you the need to manually edit the params
-file before running an experiment. It can override (`train.epochs=10`), append
-(`+train.weight_decay=0.01`), or remove (`~model.dropout`) parameters.
+file before running an experiment.
 
-<admon type="tip">
+It can override (`train.epochs=10`), append (`+train.weight_decay=0.01`), or
+remove (`~model.dropout`) parameters.
 
-DVC supports the
-[Hydra Override Grammar](https://hydra.cc/docs/advanced/override_grammar/basic/),
-with the exception of patterns that modify Hydra's defaults list.
+You can modify multiple parameters at the same time:
 
-</admon>
+```dvc
+dvc exp run -S 'prepare.split=0.1' -S 'featurize.max_features=100'
+...
+```
+
+## Example: Grid Search
```

Comment:
Added Grid Search example here as I notice the first example was using the get-started. We could actually generate the example-get-started repo based on this example:

https://github.com/iterative/example-repos-dev/blob/cfa4ca4d4097cab26b2921026dfb3a41c70fadaf/example-get-started/generate.sh#L187-L191

Reply:
Yup, great idea!

Reply:
@daavoo Do you want to create an issue or PR for that?